src/book-byen/book-byen.js

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import "./book-byen.scss";
+import { ThemeStyles } from "../slide-util";
+import GlobalStyles from "../GlobalStyles";
+
+const formatTime = (date) => {
+  return new Date(date).toLocaleTimeString("da-DK", {
+    hour: "2-digit",
+    minute: "2-digit",
+  });
+};
+
+const filterEvents = (item) => {
+  if (item?.isDeleted) {
+    // Filter the item away.
+    return false;
+  }
+  // Keep.
+  return true;
+};
```

Comment:
Can we simplify this?

```suggestion
const filterEvents = (item) => {
  return item?.isDeleted;
};
```